### PR TITLE
made footer responsive and fixed bug in like

### PR DESCRIPTION
**Pull request by aaaa760**
[] Made footer complete responsive
[] modified Extention Login Svg
[] Fixed the bug in the like functionality now the count is not incrementing on refresh

### CODE REVIEW

File: src/components/Posts/index.js [hunk 1]
```diff
@@ -294,8 +294,10 @@ function Posts(props) {
     }, [postCreatedAt]);
 
     useEffect(() => {
-        const localLikeCount = getLocalLikeCountForPost(props.note.id);
-        setVotesCount(props.note.voteCount + localLikeCount);
+        // const localLikeCount = getLocalLikeCountForPost(props.note.id);
+        // console.log(localLikeCount);
+        console.log(props.note.voteCount);
```
Comment (AaruBama) [inline src/components/Posts/index.js:299]: ??

File: src/components/Login/FooterBar.js [hunk 2]
```diff
@@ -25,7 +26,7 @@ const FooterBar = () => {
     }
 
     return (
-        <div className="fixed inset-x-0 bottom-0 bg-white border-t border-gray-400 shadow-md flex justify-around items-center md:hidden z-50">
+        <div className="footer-bar fixed inset-x-0 bottom-0 bg-white border-t border-gray-400 shadow-md flex justify-around items-center md:hidden z-50">
```
Comment (AaruBama) [inline src/components/Login/FooterBar.js:29]: do we have a class named footer-bar?

File: src/components/Posts/index.js (hunk 1 above)
Comment (aaaa760) [reply src/components/Posts/index.js]: Removed the log statements and commited the changes please check

File: src/components/Login/FooterBar.js (hunk 2 above)
Comment (aaaa760) [reply src/components/Login/FooterBar.js:29]: i created a small class in profile.css file for handling the size in very small screens using a media query so i added the class footer-bar

**Pull request merged**